examples/api-tests/src/file-search.spec.js

```diff
@@ -55,6 +57,59 @@ describe('file-search', function () {
 
         });
 
+        describe('#buildOpenerOptions', () => {
+            it('should return the correct editor opener options', () => {
+                const rangeTest = {
+                    start: { line: 3, character: 3 },
+                    end: { line: 5, character: 5 }
+                };
+                quickFileOpenService['filterAndRange'].range = rangeTest;
+                expect(quickFileOpenService['buildOpenerOptions']().selection).eq(rangeTest);
+            });
+        });
+
+        describe('#splitFilterAndRange', () => {
+
+            const expression1 = 'a:2:2';
+            const expression2 = 'a:2,2';
+            const expression3 = 'a:2#2';
+            const expression4 = 'a#2:2';
+            const expression5 = 'a#2,2';
+            const expression6 = 'a#2#2';
+            const expression7 = 'a:2';
+            const expression8 = 'a#2';
```

Comment:
Please use different values for `line` and `column` for test purposes (ex: `a:1:2`).